Makefile

```diff
@@ -88,6 +94,7 @@ controller-linux: builder
 controller-image: controller-linux
 	docker build -t $(IMAGE_PREFIX)workflow-controller:$(IMAGE_TAG) -f Dockerfile-workflow-controller .
 	@if [ "$(DOCKER_PUSH)" = "true" ] ; then docker push $(IMAGE_PREFIX)workflow-controller:$(IMAGE_TAG) ; fi
+	@if [ "$(GCP_PUSH)" = "true" ] ; then docker tag $(IMAGE_PREFIX)workflow-controller:$(IMAGE_TAG) "gcr.io/$(GOOGLE_PROJECT_ID)/workflow-controller:$(IMAGE_TAG)"; gcloud docker -- push gcr.io/$(GOOGLE_PROJECT_ID)/workflow-controller:$(IMAGE_TAG) ; fi
```

Comment:
I agree with @jessesuen. "grc.io" is part of IMAGE_PREFIX. We don't need to special treatment of it as docker push will handle it properly.